builtins/en/facets/instructions/fix.md

```
@@ -43,5 +44,10 @@ If you disputed any findings, include `## Disputed Findings` (follow the format
 | new (fixed in this iteration) | {N} |
 | reopened (recurrence fixed) | {N} |
 | persists (carried over, fixed in this iteration) | {N} |
+## Family coverage
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)